generated/nidigital/nidigital/session.py

```diff
@@ -1869,7 +2300,10 @@ def get_fail_count(self):
         nidigital.Session repeated capabilities container, and calling this method on the result.
 
         Returns:
-            failure_count (list of int):
+            failure_count (list of int): Number of failures in an array. If a site is disabled or not enabled for burst, the method does not return data for that site. If you are using a list of pin names to read data from multiple instruments, use the SortPinResultsBySiteViInt64 method to order and combine the data to match the **channelList**. You can also use the GetResultsPinMapInformation method to obtain a sorted list of returned sites and channels.
```

Comment:
"If you are using a list of pin names to read data from multiple instruments, use the SortPinResultsBySiteViInt64 method to order and combine the data to match the **channelList**."

You can get rid of this portion (SortPinResultsBySiteViInt64 is not part of public API, since multi-instrument sessions are supported).

"You can also use the GetResultsPinMapInformation method to obtain a sorted list of returned sites and channels."

GetResultsPinMapInformation --> niDigital_GetPinResultsPinInformation

Reply:
> GetResultsPinMapInformation --> niDigital_GetPinResultsPinInformation

The metadata in src/nidigital/metadata/function.py already has the 'niDigital_' prefix here, so something must be stripping it off.

Reply:
Oh, I see, it's not just the prefix.  Well, making that change just results in the name of the private python method being put in the doc string.  But, like BurstPattern, if I change it to `niDigital_FancyGetPinResultsPinInformation` then it generates the desired doc string.

Reply:
Changed to refer to Fancy function.